src/OpenTelemetry.Exporter.OneCollector/OneCollectorExporter.cs

```diff
@@ -66,16 +66,39 @@ public sealed override ExportResult Export(in Batch<T> batch)
 
     /// <summary>
     /// Register a callback action that will be triggered any time a payload is
-    /// transmitted by the exporter.
+    /// successfully transmitted by the exporter.
```

Comment:
Does this mean the callback will _only_ be called if the transmission succeeded? (the method signature indicates the other way around)

Reply:
This one yes it only fires for success which is how it works today. To receive failures you have to call the second (new) one and pass `includeFailures: true`.